osidb/helpers.py

```diff
@@ -285,3 +287,27 @@ def makeSocket(self, *args, **kwargs):
 
         result = context.wrap_socket(result, server_hostname=self.address[0])
         return result
+
+
+def bypass_rls(f: Callable) -> Callable:
+    """
+    Bypass Row-Level Security checks in the database.
+
+    When a callable is decorated with this helper, the ACLs will be set to
+    ALL_GROUPS, effectively bypassing Row-Level Security / Authorization.
+
+    Be aware of the implications of bypassing RLS, namely that any actions
+    that depend on a user's permissions will not work correctly if executed
+    within the lifetime of the callable that this decorator wraps.
+
+    Should be used with system-driven processes mostly:
+        * Migrations
+        * Collectors & Scheduled tasks
+        * Non user-driven processes
+    """
+
+    def wrapped(*args, **kwargs):
+        set_user_acls(settings.ALL_GROUPS)
+        f(*args, **kwargs)
+
+    return wrapped
```

Comment:
Based on the description I would expect the ACLs to be unset to the previous/no ACLs after the call but this seems to simply retain the full access.

Reply:
In theory, yes, but it doesn't happen in practice and this is the pattern we use all over OSIDB.

That said, I'll change it as a sanity check

Reply:
That is why I am not insisting on the extension, I am just pointing out that the wrapper description says something little bit different than it is actually done then